### DISCUSSION TITLE
Void Linux install to VirtualBox virtual machine fails with error

### DISCUSSION THREAD

**Discussion by tygertechforce** [ipxe/ipxe#805] [Q&A]
The Void Linux .iso file was downloaded from their official website [https://voidlinux.org/download/](url).

I went through the entire setup process for the VM, but when I "Start"ed the VM I got the following error (which doesn't seem to be documented where I could find the solution, at least not in my web searching):

     > tftp://10.0.x.x/Void%20Linux.pxe... Permission denied (http://ipxe.org/0212603c)

The install is halted at a "No more network devices" message (after it previously successfully completed finding and linking to my ethernet connection (net0)).

Above in the script it displays this:

     [Link status: Down (http:ipxe.org/38086101)]

So I am not sure if the file it's trying to access to continue the install is invalid, unavailable, or other. As such, I am unable to install Void Linux into a VM.

Any comments, help, and suggestions are most welcomed. And if anyone has a working Void Linux..vdi file or knows where I can get it, please let me know.

Thanks in advance!

![VoidLinux_VM-Install-Error](https://user-images.githubusercontent.com/14300315/202392967-7c610a7a-a0f8-4398-a80a-af67287f5e47.jpg)

**Comment by NiKiZe** [comment]
Link from message http://ipxe.org/0212603c

Permission denied from tftp server, check your tftp server logs.
All this is explained at that page.

Since this is unlikely to be an issue with iPXE it was moved to discussions.

**Reply by tygertechforce** [reply]
Thanks for the reply. I don't use tftp and don't have any logs. I also don't use iPXE.  I am simply trying to get a virtual machine made for Void Linux.

However, I did find the solution on this webpage: [https://linuxiac.com/void-linux-installation/](url)

> VirtualBox’s and KVM’s virt-manager users must enable EFI in the virtual machine settings before starting the installation. Otherwise, an error will occur when attempting to install the GRUB bootloader.

I really hope this helps someone else who came across this error message. Mine might be a unique situation with no one else trying this, but who knows.

Thanks for the help!

**Reply by hadi9890** [reply]
Just encountered the same issue while installing Manjaro GNOME, the solution worked like a charm!

**Reply by EccoArts** [reply]
tnx

**Comment by XLEXFERR02** [comment]
Holaa.
Mil gracias, el link ya no funcioa, pero con lo que escribiste fue suficiente y lo pude instalar. Mil gracias men

**Comment by AdmPro007** [comment]
hola, tuve el mismo  problema, descargue **_VM VirtualBox 7.xxx Oracle VM VirtualBox Extension Pack_**, son drivers para maquina virtual. instale y a corrido de maravilla. 
se encuentra en la misma pagina _https://www.virtualbox.org/wiki/Downloads_
